src/scripts/accesskit.css

```
@@ -52,6 +52,7 @@ figure.accesskit-visible-alt-text {
 
 figure.accesskit-visible-alt-text figcaption {
   height: 100%;
+  min-width: min-content;
```

Apparently the true number of pixels for the width of a single image in a three-image row is `177.33334350585938px`. Also, setting that without accounting for the padding breaks three-image photoset rows!

Beyond that, I realised that using a pixel value (or a `calc()` assuming the post width to be `540px` ) would break on viewport widths under 540px wide. I thought we might need to use a media query, but for once, CSS `min()` looks like it actually does its job!

```suggestion
  box-sizing: border-box;
  height: 100%;
  min-width: calc((min(540px, 100vw) / 3) - (4px * 2));
```

---

I... cannot believe I didn't know about `box-sizing: border-box;` :D

---

Ah, I see what you're doing here: setting a minimum size equal to the smallest an image could be in a photoset.